api/tests/integration/ref/formats/mol_to_cdxml.py.out

```diff
@@ -19,49 +20,49 @@ enhanced_stereo1.mol
         <color r="0.5" g="0.5" b="0.5"/>
     </colortable>
     <page HeightPages="1" WidthPages="1">
-        <fragment>
-            <n id="1" Element="8" NumHydrogens="1" p="30.000000 30.000000">
+        <fragment id="3">
+            <n id="4" Element="8" NumHydrogens="1" p="30.000000 30.000000">
                 <t p="30.000000 30.000000" LabelJustification="Left">
                     <s font="3" size="10" face="96">OH</s>
                 </t>
             </n>
-            <n id="2" Element="6" p="70.011002 53.099998"/>
-            <n id="3" Element="6" p="70.011002 99.299995" Geometry="Tetrahedral" BondOrdering="2 4 11 0" EnhancedStereoType="Absolute"/>
-            <n id="4" Element="6" p="110.021996 122.399994"/>
-            <n id="5" Element="6" p="110.021996 168.599991"/>
-            <n id="6" Element="6" p="150.029999 191.699997"/>
-            <n id="7" Element="6" p="150.029999 237.899994"/>
-            <n id="8" Element="6" p="110.021996 261.000000"/>
-            <n id="9" Element="6" p="70.011002 237.899994"/>
-            <n id="10" Element="6" p="70.011002 191.699997"/>
-            <n id="11" Element="7" NumHydrogens="1" p="30.000000 122.399994">
+            <n id="5" p="70.011002 53.099998"/>
+            <n id="6" p="70.011002 99.299995" Geometry="Tetrahedral" BondOrdering="5 7 14 4" EnhancedStereoType="Absolute"/>
+            <n id="7" p="110.021996 122.399994"/>
```

Comment:
I can see that Element was removed. Just double check - is it expected

Reply:
fixed